### ISSUE TITLE
fastmod

### ISSUE THREAD

**Issue by tk90** [kolton/d2bot-with-kolbot#607]
So.. is fastmod working?
and packet casting?

should i use both together? or do i up my chance of getting banned?

i tried it out but i dont feel like it is working...

thanks for your answers!

**Comment by gogos-venge** [comment]
@DarkHorseDre true, packet shopping is not an exploit and shouldn't be detectable (at least by the server). I updated my response for more info #599

**Comment by Druxlolz** [comment]
I have been using Packet Casting for my Hammerdins for a long time.
No problems at all running, 1.5 month smooth.

**Comment by tk90** [comment]
@DarkHorseDre hey sorry for the late answer, just came back from a trip!

so i just used the fastmod on one diablo 2 and watched from a second d2 window if it worked (the one in the char config). it didnt seem to work like whenever i teleported it teled on the spot a lot and when i used skills it looked as if i had a lot of fcr but only from the d2 with the fast mod activated.

others would just see the char without fcr it seems..

i hope i didnt mess up my explanation too much xD
thanks for all your input and help guys!

**Comment by DarkHorseDre** [comment]
haha no worries bro

ive never used it but from the description i imagine that's how it works:  instead of client-side actions sending data to server for it to react to, the packets are sent direct with no client side changes. so the animations and other client stuff wouldn't happen, or at least not happen at the rate the server see's.
